README.md

```diff
@@ -41,7 +41,9 @@ The description of the input parameters that can go in the run command is as fol
 8. **metamodel-components**: If this parameter is passed, then each metamodel component retreived from the vCenter server is saved in a different .json file under the metamodel directory.
 9. **host**: It is the IP Address of the host that serves the API. By default the value is < vcenter >
 10. **oas** : This parameter is used to specify as to which version of swagger file the user wants to generate. By default the generated files are of version 3 i.e openapi. If the user wants to generate the version 2 files, the parameter needs to be passed explicitly.
-11. **deprecate-slash-rest** : This parameter is used to deprecate the /rest APIs in the generated OpenAPI specification, only when the API to deprecate has an /api counterpart.
+11. **deprecate-slash-rest**: This parameter is used to deprecate the /rest APIs in the generated OpenAPI specification, only when the API to deprecate has an /api counterpart.
+12. **fetch-authentication-metadata**: Authentication information per operation is retrieved via the authentication metadata service, which contributes to the total generation time. This parameter is used to specify whether the authentication information is going to be retrieved. 
+13. **auto-rest-services**: The parameter is used to acquire the 6.0 Auto Rest services. Following the parameter are space separated service IDs.
```

Comment:
this is going to be a pretty long list, are we safe about it? I mean the cmd line which we use to start a process. There are some notable restrictions on Windows, but we should make sure we are fine on Linux.

Please also explain that this parameter is optional, and what is the benefit of passing it (getting faster generation, I believe, if the tool doesn't have to discover the services on its own).

Reply:
It seems that the cmd character limit is 8191 characters (https://support.microsoft.com/en-us/help/830473/command-prompt-cmd-exe-command-line-string-limitation) - the passed command is roughly 2000 characters, hence it should not be a problem for windows (assuming for linux as well - it will be tested with the endpoint integration soon) 

I decided to remove this parameter, because (as per our offline discussion) the open-source users are not aware which are the auto-rest-services and most likely only the vapi-endpoint will utilize this parameter.